content/en/blog/falcoctl-install-manage-rules-plugins.md/index.md

```diff
@@ -0,0 +1,383 @@
+
+- `Artifact`: An artifact is an element `falcoctl` can manipulate, right now, only `rules` and `plugins` are considered.
+- `Index`: A .yaml file containing a list of available artifacts with their registries and repositories. An automatic generic index is provided for official artifacts, see [here](https://github.com/falcosecurity/falcoctl/tree/gh-pages).
+- `Registry`: A registry stores the artifacts, that `falcoctl` understands in regards to the [OCI standard](https://opencontainers.org/), any compliant OCI can be used. The official registry uses [Github Packages](https://github.com/orgs/falcosecurity/packages).
+- `Repository`: similar to containers, it belongs to a registry and contains one or more versions (tags) of an artifact.
+
+## Use falcoctl locally
+
+The following steps explain the basic concepts used to install `falcoctl` locally.
+
+### Installation
+
+For a Linux amd64 system:
+```shell
+LATEST=$(curl -sI https://github.com/falcosecurity/falcoctl/releases/latest | awk '/location: /{gsub("\r","",$2);split($2,v,"/");print substr(v[8],2)}')
+curl --fail -LS "https://github.com/falcosecurity/falcoctl/releases/download/v${LATEST}/falcoctl_${LATEST}_linux_amd64.tar.gz" | tar -xz
+sudo install -o root -g root -m 0755 falcoctl /usr/local/bin/falcoctl
+```
+
+> Other environments are described in the [README](https://github.com/falcosecurity/falcoctl#install-falcoctl-manually)
+
+Now you are able to confirm that the installation went well.
+Check the version:
+```shell
+❯ falcoctl version
+
+Client Version: 0.4.0
+```
+Print the help:
+```shell
+❯ falcoctl --help
+
+     __       _                _   _
+    / _| __ _| | ___ ___   ___| |_| |
+   | |_ / _  | |/ __/ _ \ / __| __| |
+   |  _| (_| | | (_| (_) | (__| |_| |
+   |_|  \__,_|_|\___\___/ \___|\__|_|
+									
+	
+The official CLI tool for working with Falco and its ecosystem components.
+
+Usage:
+  falcoctl [command]
+
+Available Commands:
+  artifact    Interact with Falco artifacts
+  completion  Generate the autocompletion script for the specified shell
+  help        Help with any command
+  index       Interact with index
+  registry    Interact with OCI registries
+  tls         Generate and install TLS material for Falco
+  version     Print the falcoctl version information
+
+Flags:
+      --config string     config file to be used for falcoctl (default "/etc/falcoctl/falcoctl.yaml")
+      --disable-styling   Disable output styling such as spinners, progress bars and colors. Styling is automatically disabled if not attached to a tty (default false)
+  -h, --help              help for falcoctl
+  -v, --verbose           Enable verbose logs (default false)
+
+Use "falcoctl [command] --help" for more information about a command.
+
+```
+
+### Index
+
+Before being able to download and install artifacts, we need to configure an index, we provide one for all [official artifacts](https://falcosecurity.github.io/falcoctl/index.yaml), plugins and rules:  
```

Comment:
Not completely true in my opinion: if you want to install and follow artifacts, the tool has its own default config containing our official index. On the other hand, this section could be used to show how a custom index can be configured, but normal users will never use this.

Reply:
The configuration and setup of a custom index is for a next blog post.

Reply:
Yeah so maybe I was thinking we can skip this `Index` section?

Reply:
I see that as an introduction to make people understand the concepts.